__tests__/commands/_helpers.js

```diff
@@ -23,7 +23,7 @@ export const runInstall = run.bind(
   async (args, flags, config, reporter, lockfile): Promise<Install> => {
     const install = new Install(flags, config, reporter, lockfile);
     await install.init();
-    await check(config, reporter, {}, []);
+    await check(config, reporter, flags, []);
```

Comment:
I don't think we should pass flags for install and check via the same variable.
Can you please revert this change?